src/GridFS/BucketReadWriter.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace MongoDB\GridFS;
+
+class BucketReadWriter
```

Comment:
This class still exists, but is no longer called from anywhere. Just keeping it around until we're sure we're done with it.